tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -522,6 +522,100 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 					})
 				})
 
+				When("a Redis instance is created with a specific name and json output", func() {
+
+					var name string
+					var svcFullName string
+					var output string
+
+					BeforeEach(func() {
+						name = helper.RandString(6)
+						svcFullName = strings.Join([]string{"Redis", name}, "/")
+						output = helper.Cmd("odo", "service", "create", fmt.Sprintf("%s/Redis", redisOperator), name, "--project", commonVar.Project, "-o", "json").ShouldPass().Out()
+					})
+
+					AfterEach(func() {
+						helper.Cmd("odo", "service", "delete", svcFullName, "-f").ShouldRun()
+					})
+
+					It("should display valid information in output of create command", func() {
+						By("displaying service information", func() {
+							values := gjson.GetMany(output, "kind", "metadata.name", "manifest.kind", "manifest.metadata.name")
+							expected := []string{"Service", "Redis/" + name, "Redis", name}
+							Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+						})
+
+						By("not containing cluster specific information", func() {
+							values := gjson.GetMany(output, "inDevfile", "deployed", "manifest.metadata.creationTimestamp")
+							expected := []string{"true", "false", ""}
+							Expect(helper.GjsonMatcher(values, expected)).To(Equal(true))
+						})
```

Comment:
Do you think we can move these `By` statements to a local function to reduce the duplicity?
Something like https://github.com/openshift/odo/pull/4927/files#diff-7373622ac627f4f7b03b7aac291951160b62c57c045121d79c3839e625c19285R136 ?

I would like to hear your reasons if you think otherwise. :)

Reply:
Yes you are right, I'll make the change